src/Servers/IIS/AspNetCoreModuleV2/InProcessRequestHandler/inprocessapplication.cpp

```diff
@@ -241,25 +241,19 @@ IN_PROCESS_APPLICATION::ExecuteApplication()
             LOG_INFOF(L"Setting current directory to %s", this->QueryApplicationPhysicalPath().c_str());
         }
 
-        if (m_pConfig->QueryCallStartupHook())
+        auto startupReturnCode = context->m_hostFxr.InitializeForApp(context->m_argc, context->m_argv.get(), m_dotnetExeKnownLocation);
+        if (startupReturnCode != 0)
         {
-            // Used to display developer exception page when there is an exception in main.
-            auto currentStartupHookEnv = Environment::GetEnvironmentVariableValue(DOTNETCORE_STARTUP_HOOK);
```

Comment:
What happened to this?

Reply:
It's still there: https://github.com/aspnet/AspNetCore/pull/10121/files#diff-ba7e6a78e2d291ac7cfef69259096922R252?

Reply:
But you were adding before and now you aren't?

Reply:
We are still setting the STARTUP_HOOK to a value, we are just using a runtime property instead of the environment variable. We don't need to append the startup hook as an environment variable anymore as we are using runtime properties.